src/test/py/server/new_server_test.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+"""
+Test Module for the Protocol class
+"""
+
+import unittest
+import asyncio
+import hashlib
+
+from unittest.mock import Mock
+from server.network import NewServer
+
+
+class TestNewServer(unittest.IsolatedAsyncioTestCase):
+    """Test class for NewServer class"""
+
+    def setUp(self):
+        """Create instances of NewServer"""
+        self.mock_store_callback = Mock()
+
+        self.node1 = NewServer(self.mock_store_callback)
+        self.node2 = NewServer(self.mock_store_callback)
+
+    def test_init(self):
+        """Test the __init__ method of NewServer"""
+        self.assertEqual(self.node1.store_callback, self.mock_store_callback)
+
+    async def test_store(self):
+        """Test the store method of NewServer"""
+        await self.node1.listen(8468)
+        await self.node2.listen(8469)
```

Comment:
Please use ports in the dynamic range: 49152 - 65535.

See: https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.txt